styles/globalStyles.js

```diff
@@ -213,11 +213,11 @@ export const MyTextInput = ( { label, icon, ...props }) => {
   );
 };
 
-export const PasswordTextInput = ( { hidePassword, setHidePassword, ...props }) => {
+export const PasswordTextInput = ( { Text, hidePassword, setHidePassword, ...props }) => {
   const touchRef = useRef();
   return (
       <View>
-        <Text>Password</Text>
+        {Text}
```

Comment:
you can wrap this in `<Text>{Text}</Text>`

Reply:
Also if you changed this you probably need to pass in a value for login.js

Reply:
> you can wrap this in `<Text>{Text}</Text>`

I tried but didn't work

Reply:
> Also if you changed this you probably need to pass in a value for login.js

True, I forgot about that one

Reply:
I think it's because you capitalized Text